app/models/set_member_subject.rb

```diff
@@ -4,7 +4,6 @@ class SetMemberSubject < ActiveRecord::Base
   
   belongs_to :subject_set, counter_cache: true, touch: true
   belongs_to :subject
-  has_many :classifications
```

Comment:
Storm/Kafka integration is part of the Storm project, and it automatically handles kafka failures and offset tracking. 

Let me knock out a demo of the storm topology (I think it would take less than a day), and we can figure it out from there.